itg-tests/es-it/ReeferItgTests.yaml

```diff
@@ -15,13 +15,26 @@ spec:
         - /bin/bash
         - -c
         - |
+              # Understand what environment we are executing against
+              if [ -z "$KAFKA_USER" ]; then
+                KAFKA_ENV="LOCAL"
+              elif [ "$KAFKA_USER" == "token" ]; then
+                KAFKA_ENV="IBMCLOUD"
+              else
+                KAFKA_ENV="OCP"
+              fi
+
+              # Export KAFKA_ENV since it is used by the test cases
+              export KAFKA_ENV=${KAFKA_ENV}
+
               echo "-----------------------------------------------------------------"
               echo "-- Reefer Container Shipment EDA application Integration Tests --"
               echo "-----------------------------------------------------------------"
               echo
               echo "Executing integrations tests from branch $BRANCH of $GITHUB_REPO"
               echo "Kafka Brokers: $KAFKA_BROKERS"
-              echo "Kafka API Key: $KAFKA_APIKEY"
+              echo "Kafka user: $KAFKA_USER"
+              echo "Kafka password: $KAFKA_PASSWORD"
```

Comment:
Let's obfuscate the password here. Something like first letter + "*******" + last letter.  We can understand we're using the correct password looking back, but it isn't exposing the Kafka Credentials explicitly.

Reply:
If that is too much to work through in just BASH, it can just be "*****" or "NULL" depending on values...

Reply:
Just committed some code to obfuscate the password. Let me know what you think